crates/swc_plugin_backend_wasmer/Cargo.toml

```
@@ -0,0 +1,68 @@
+[package]
+name = "swc_plugin_backend_wasmer"
+version = "0.1.0"
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
```

```suggestion
edition = { workspace = true }
license = { workspace = true }
repository = { workspace = true }
```

It's a bug of `cargo-mono`, but this syntax is not supported.